cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -2486,6 +2493,262 @@
          {:align-form-columns? true
           :aligned-forms {'foobar #{0}}}))))
 
+(deftest test-align-form-columns-with-comments
+  (testing "comments at end of bindings should not affect alignment"
+    (is (reformats-to?
+         ["(let [x 1   ; a comment"
+          "      longer 2]"
+          "  (+ x longer))"]
+         ["(let [x      1   ; a comment"
+          "      longer 2]"
+          "  (+ x longer))"]
+         {:align-form-columns? true})))
+  (testing "comments between bindings should not affect alignment"
+    (is (reformats-to?
+         ["(let [binding1 \"value1\""
+          "      ;; This is a comment"
+          "      binding2 \"value2\""
+          "      ;; Another comment"
+          "      longer-binding \"value3\"]"
+          "  (+ binding1 binding2))"]
+         ["(let [binding1       \"value1\""
+          "      ;; This is a comment"
+          "      binding2       \"value2\""
+          "      ;; Another comment"
+          "      longer-binding \"value3\"]"
+          "  (+ binding1 binding2))"]
+         {:align-form-columns? true}))
+    (is (reformats-to?
+         ["(let [a 1"
+          "      ;; comment"
+          "      bb 2"
+          "      ccc 3]"
+          "  (+ a bb ccc))"]
+         ["(let [a   1"
+          "      ;; comment"
+          "      bb  2"
+          "      ccc 3]"
+          "  (+ a bb ccc))"]
+         {:align-form-columns? true})))
+  (testing "let starting with long comment should not affect alignment"
+    (is (reformats-to?
+         ["(let [;; This is a very long comment at the beginning of the let"
+          "      a 1"
+          "      bb 2"
+          "      ccc 3]"
+          "  (+ a bb ccc))"]
+         ["(let [;; This is a very long comment at the beginning of the let"
+          "      a   1"
+          "      bb  2"
+          "      ccc 3]"
+          "  (+ a bb ccc))"]
+         {:align-form-columns? true}))))
+
+(deftest test-align-form-columns-edge-cases
```

Comment:
I don't understand how this set of tests is related to the feature the PR is implementing.

As a general rule, the tests added in this PR should fail on master, otherwise they're not testing the feature. This isn't a hard rule - there are exceptions - but it's a decent rule of thumb,